requirements.txt

```diff
@@ -32,4 +32,5 @@ urllib3==1.23
 wrapt==1.10.11
 flask
 fbmessenger
-dialogflow==1.1.1
\ No newline at end of file
+dialogflow==1.1.1
+google-cloud-bigquery
```

Comment:
@owem0410 can we specify the version for bigquery?

Reply:
Ok, I will specify “2.24”